src/Components/Publishing/Series/Series.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component  } from "react"
+import styled, { StyledFunction } from "styled-components"
+import { pMedia } from "../../Helpers"
+import { ArticleCard } from './Components/ArticleCard'
+import { SeriesAbout } from './Components/SeriesAbout'
+import { SeriesTitle } from './Components/SeriesTitle'
+
+interface Props {
+  color?: string,
+  series?: any
+}
+
+export class Series extends Component<Props, null> {
+  public static defaultProps: Partial<Props>
+
+  render () {
+    const { color, series } = this.props
+    const articles = series.related_articles || []
+
+    return (
+      <SeriesContainer className='SeriesTitle' color={color}>
+        <SeriesContent>
+
+          <SeriesTitle
+            series={series}
+            color={color}
+          />
+          
+          {articles.map((article, i) =>
+            <ArticleCard
+              key={i}
+              article={article}
+              series={series}
+              color={color}
+            />
+          )}
+          
+          <SeriesAbout
+            series={series}
+            color={color}
+          />
+
+        </SeriesContent>
+      </SeriesContainer>
+    )
+  }
+}
+
+Series.defaultProps = {
+  color: 'white'
+}
+
+const Div: StyledFunction<Props & React.HTMLProps<HTMLDivElement>> = styled.div
+
+const SeriesContainer = Div`
+  color: ${props => props.color};
+  background-color: black;
+  padding: 150px 20px;
+`
+const SeriesContent = styled.div`
+  max-width: 1200px;
+  margin: 0 auto;
+
+  .SeriesTitle {
+    margin-bottom: 60px;
+    .PartnerBlock {
+      padding-bottom: 40px;
+    }
+  }
+
+  .ArticleCard {
+    margin-bottom: 60px;
+  }
+
+  .SeriesAbout {
+    padding-top: 60px;
+  }
```

Comment:
Same here, we should be able to reference and update with: 

```
${SeriesAbout} {
   padding-top: 60px;
}
```
to avoid usage of className.

Reply:
Doesn't that only work for targeting a styled component?  These are references to react components nested in the series.  Didn't want to include the outside padding in the component itself because it could potentially change in the future based on context.

Reply:
Yes, it's for targeting a styled component. I'm saying we could replace the usage of `className="SeriesAbout"` and make an exportable styled component called `<SeriesAbout>` to reference it like above. Orr we could even pass in margin as a prop?